standalone/pruntime/enclave/src/system/gk.rs

```diff
@@ -607,15 +607,15 @@ where
             if let Some(master_key) = &self.state.master_key {
                 info!("Gatekeeper: try dispatch master key");
                 let derived_key = master_key
-                    .derive_secp256k1_pair(&[b"master_key"])
+                    .derive_sr25519_pair(&[b"master_key"])
```

Comment:
Why the derived key is by keyword "master_key"?

Reply:
fixed